backend/schema/corpora_schema.md

```diff
@@ -0,0 +1,323 @@
+
+**Field name**|**Constraints**
+:--|:--
+organism|String
+organism\_ontology|NCBITaxon term
+layer\_descriptions|Mapping from {layer\_name: layer\_description, …} Each description is free text, though one layer must be described as “raw”.
+
+As with the ontology fields above, the ontology is added as part of Corpora curation.
+
+
+#### Presentation Metadata
+
+There are also fields that are required so that Corpora (both Portal and cellxgene) can present datasets appropriately.
+
+Each dataset must have at least one **embedding**, a mapping from each cell to a pair of floats. These are usually generated by algorithms like umap or
+tsne and are used to display the dataset in cellxgene. If a submitted dataset does not have an embedding, the umap embedding will be calculated and inserted
+during curation.
+
+Datasets also must have a few other metadata fields for presentation:
+
+**Field name**|**Description**
+:--|:--
+title|String that identifies the dataset
+contributors|A list of {“name”: …, “institution”: …, “email”: …}, where each value is a string. Only “name” is required.
+
+
+If the dataset has an associated preprint or journal publication, these fields are required:
+
+**Field name**|**Description**
+:--|:--
+preprint\_doi|DOI of the associated preprint.
+publication\_doi|DOI of the associated journal publication.
+
+
+#### Presentation Hints
+
+The metadata fields below are fully optional. They aren’t needed for integration, and Corpora can present the data fine without them, but if they are
+present Corpora will do something with them. This allows submitters to fine-tune how their datasets are presented, which is a common request.
+
+
+<table>
+  <tr>
+   <td><strong>Field name</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>color_map
+   </td>
+   <td>Submitters can include a field called “{field}_colors” for any other categorical integer metadata field. The value must be an array of one of
+       fourcolor specifications:
+<ul>
+
+<li>CSS4 color name, as supported by matplotlib
+    <a href="https://matplotlib.org/3.1.0/gallery/color/named_colors.html">https://matplotlib.org/3.1.0/gallery/color/named_colors.html</a>
+
+<li>RGB tuple/list with values ranging from 0.0 to 1.0, as in [0.5, 0.75, 1.0]
+
+<li>RFB tuple/list with values ranging from 0 to 255, as in [128, 192, 255]
+
+<li>Hex triplet string, as in "#08c0ff"and each string must be a hex color code.
+
+</li>
+</ul>
+The color code at the nth position in the array corresponds to category n in the metadata field.
+
+   </td>
+  </tr>
+  <tr>
+   <td>tags
+   </td>
+   <td>A list of strings that the Portal or other tools may use for searching, sorting, or filtering.
+   </td>
+  </tr>
+  <tr>
+   <td>default_field
+   </td>
+   <td>Name of another field that should be the “default” for display. So for example, when cellxgene starts this is the field that the cells should be
```

Comment:
missing comma:   `...starts, this ...`

Reply:
It feels like a lot of commas to me, but I will add.

Reply:
eats, shoots, and leaves...?

Reply:
So, for example, when, cellxgene starts, this is, the field, that .. <- do that. Much more dramatic.

Reply:
So 👏 for 👏 example 👏 when 👏 cellxgene 👏 starts 👏